tests/test_project/admin.py

```diff
@@ -29,6 +30,6 @@ class ProjectAdmin(admin.ModelAdmin):
     list_display = ['name']
 
 
-admin.site.register(Operator, OperatorAdmin)
-admin.site.register(RadiusAccounting, RadiusAccountingAdmin)
-admin.site.register(Project, ProjectAdmin)
+admin_site.register(Operator, OperatorAdmin)
```

Comment:
should be done this way: https://docs.djangoproject.com/en/3.0/ref/contrib/admin/#django.contrib.admin.register